apps/vocabulary/main.go

```diff
@@ -144,7 +54,25 @@ func main() {
 	var properties map[string]int
 	properties = make(map[string]int)
 
-	processDocument(document, schemas, operationId, names, properties)
+	//Temporary, for now using filename to check file type
+	version_flag := strings.Contains(args[0], "swagger")
```

Comment:
LGTM. To other reviewers: we're planning to migrate this code into a plugin, and when we do, the plugins framework will tell us what version of spec we're getting so we won't need to switch on anything arbitrary.